### PR TITLE
Shameless rider kit buff (and to some degree, beastmaster buff)

### PR DESCRIPTION
**Pull request by RinPin**
adds a capture device to the rider kit, so it's a lot easier to have your own pet chocobo / motorcycle without them constantly dying due to ghouls attacking them or getting stolen due to the lack of keys, buffs beastmaster by adding an additional capture egg to the kit, to hopefully encourage more diversity in battle pets, (i'll probably make an eggless version of the rider kit for cargo if people abuse this)

## Pre-Merge Checklist
- [Y ] You tested this on a local server.
- [ Y] This code did not runtime during testing.
- [ Y] You documented all of your changes.

### CODE REVIEW

**Comment by RinPin** [comment]
Oh god linter BS, I’ll solve this when I get home

**Comment by Tk420634** [comment]
Beat it with a rock, rin

File: code/game/objects/items/loadout_toolkits.dm [hunk 1]
```diff
@@ -578,6 +579,7 @@
 
 /obj/item/storage/box/tools/beastmaster/PopulateContents()
 	new /obj/item/capturedevice(src)
+		/obj/item/capturedevice(src)
```
Comment (DameonOwen) [inline code/game/objects/items/loadout_toolkits.dm:582]: `new` in front if this

File: code/game/objects/items/loadout_toolkits.dm [hunk 2]
```diff
@@ -60,8 +60,9 @@
 	new /obj/item/brahminbridle(src)
 	new /obj/item/brahminsaddle(src)
 	new /obj/item/brahminbrand(src)
+		/obj/item/capturedevice(src)//hopefully this will make having a pet motorcycle a lot less annoying
```
Comment (DameonOwen) [inline code/game/objects/items/loadout_toolkits.dm:63]: `new` in front of this

File: code/game/objects/items/loadout_toolkits.dm [hunk 3]
```diff
@@ -60,8 +60,9 @@
...
 	new /obj/item/choice_beacon/pet/mountable(src)
-
+        /object
```
Comment (DameonOwen) [inline code/game/objects/items/loadout_toolkits.dm:65]: Remove this line

**Review by DameonOwen — CHANGES_REQUESTED** [review]

**Review by DameonOwen — APPROVED** [review]

**Pull request merged**